app/Http/Requests/UpdateInventory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Traits\ApiRequest;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateInventory extends FormRequest
+{
+    use ApiRequest;
+
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
```

Comment:
Would be cleaner to make this inherit from StoreInventory and remove the items.* stuff and add the "item_id" property instead. THat way when we add new stuff with the redesign, we don't need to duplicate the validation